app/src/main/java/com/framgia/fsalon/screen/homestylist/HomeStylistViewModel.java

```diff
@@ -0,0 +1,106 @@
+package com.framgia.fsalon.screen.homestylist;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.support.annotation.IntDef;
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.app.Fragment;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
+
+import com.android.databinding.library.baseAdapters.BR;
+import com.framgia.fsalon.R;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.framgia.fsalon.screen.homestylist.HomeStylistViewModel.Tab.TAB_BOOKING;
+
+/**
+ * Exposes the data to be used in the Homestylist screen.
+ */
+public class HomeStylistViewModel extends BaseObservable implements HomeStylistContract.ViewModel {
+    private HomeStylistContract.Presenter mPresenter;
+    private AppCompatActivity mActivity;
+    private StylistHomePagerAdapter mAdapter;
+    private int mTabPosition;
+    private BottomNavigationView.OnNavigationItemSelectedListener mListener =
+        new BottomNavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                switch (item.getItemId()) {
+                    case R.id.item_booking:
+                        setTabPosition(Tab.TAB_BOOKING);
```

Comment:
WARNING: [UnnecessaryFullyQualifiedName] Unnecessary use of fully qualified name 'Tab.TAB_BOOKING' due to existing static import 'com.framgia.fsalon.screen.homestylist.HomeStylistViewModel.Tab.TAB_BOOKING'
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#UnnecessaryFullyQualifiedName